.github/workflows/pyomslnxwheel.yml

```diff
@@ -45,7 +45,7 @@ jobs:
 
     - name: Load contrib build
       run: |
-          curl -o contribbld.tar.gz https://abibuilder.informatik.uni-tuebingen.de/archive/openms/contrib/windows/x64/msvc-15/contrib_build.tar.gz
+          curl -o contribbld.tar.gz https://abibuilder.informatik.uni-tuebingen.de/archive/openms/contrib/windows/x64/msvc-14.2/contrib_build.tar.gz
```

Comment:
so contrib is build with 14.2 and this is build with 16?

Reply:
no, I dont know where CMake takes their numbers from. They probably just internally increase numbers now. But default toolchain in VS2019 is msvc14.2

Reply:
I recently switched the labeling of the contrib builds to the actual compiler version, so it is easier to see which are compatible.

Reply:
contrib = VS 2019 x64 (?) 
github env = Windows Server 2019 (VS 2019)
qt = win64_msvc2017_64 

I guess that is not going to work anyhow.

Reply:
Haha confusing. Server2019 is the name of the OS.
And VS2017 and 2019 are compatible. But yeah, we should change to Qt for vs2019 if already provided by the Qt Company (they are always a bit late)

Reply:
as far as I have seen https://github.com/jurplel/install-qt-action
It is not yet available (newest: win64_msvc2017_64), but that should work / compatibility wise?